README.md

```diff
@@ -1,215 +1,110 @@
+## Overview
 
-You can skip this process if you have access to a snapshot of the blockchain after height 6910000. You can find snapshots at [cosmos.quicksync.io](https://cosmos.quicksync.io/).
+The current Gaia Version of the Cosmos Hub mainnet is [`v6.0.0`](https://github.com/cosmos/gaia/releases/tag/v6.0.0). To bootstrap a mainnet node, it is possible to sync from `V6.0.0` via Quicksync or via [State Sync](https://hub.cosmos.network/main/hub-tutorials/join-mainnet.html#state-sync).
 
-# Join the Cosmos Hub Mainnet
+For a full set of instructions on boostrapping a mainnet node, see the Hub's [**Join the Cosmos Hub Mainnet**](https://hub.cosmos.network/main/hub-tutorials/join-mainnet.html) documentation.
 
-## Quickstart
-
-**Instant Gratification Snippet**
-
-```bash
-git clone -b v4.2.0 https://github.com/cosmos/gaia
-cd gaia
-make install
-gaiad init chooseanicehandle
-wget https://github.com/cosmos/mainnet/raw/master/genesis.cosmoshub-4.json.gz
-gzip -d genesis.cosmoshub-4.json.gz
-mv genesis.cosmoshub-4.json ~/.gaia/config/genesis.json
-gaiad start --p2p.seeds bf8328b66dceb4987e5cd94430af66045e59899f@public-seed.cosmos.vitwit.com:26656,cfd785a4224c7940e9a10f6c1ab24c343e923bec@164.68.107.188:26656,d72b3011ed46d783e369fdf8ae2055b99a1e5074@173.249.50.25:26656,ba3bacc714817218562f743178228f23678b2873@public-seed-node.cosmoshub.certus.one:26656,3c7cad4154967a294b3ba1cc752e40e8779640ad@84.201.128.115:26656 --x-crisis-skip-assert-invariants
-```
-Now wait until the chain reaches block height XXXX. It will panic and you will see a message like:
-```
-ERR UPGRADE "Gravity-DEX" NEEDED at height: 6910000: v5.0.0-4760cf1f1266accec7a107f440d46d9724c6fd08
+However to build a node from scratch a node operator will need to first run [v4.2.6](https://github.com/cosmos/gaia/releases/tag/v4.2.6) until the node panics at block height [6910000](https://github.com/cosmos/gaia/blob/main/docs/migration/cosmoshub-4-delta-upgrade.md#Upgrade-will-take-place-July-12,-2021). The node should stop running after the panic, if it does not stop automatically, wait for 5-10 minutes and then kill it manually. Then install the latest version of gaia ([v5.0.2](https://github.com/cosmos/gaia/releases/tag/v5.0.2)) and then begin running the binary agian with the optional flag `--x-crisis-skip-assert-invariants`. This will begin syncing the node since the last upgrade until it is at the current height.
```

Comment:
the "however to build a node from scratch," seems like a suggestion that people might not want to follow? or at least why would you do this rather than do some other (potentially less complicated?) method? 

Also following this paragraph, involves running old software, waiting for a panic and maybe killing it if it doesn't die after the panic, though I suspect that manual kill operation is less common. I might break this down the steps into the bullet points for the summary and omit the edge cases from the summary and cover them in the actual context of the step.

Reply:
i would actually just remove this whole paragraph. it's inaccurate and redundant with the contents of the "join the cosmos hub mainnet" link above.

Reply:
```suggestion
```